en/cpp/api_changes.md

```diff
@@ -21,9 +21,91 @@ This means that breaking changes to the API result in a bump of the major versio
   We plan to release version 1.0.0 in the near future, after which the above strategy will be adopted (i.e. currently breaking changes can occur in both minor and patch releases).
 
 
+## v0.33.0
+
+There are some changes in how systems are discovered, connected and accessed with this release.
+The previous APIs are still available but marked deprecated and they will be removed in the future.
+The deprecations are also implemented as compiler warnings.
+
+### Discovery of systems
+
+The 64bit (8 bytes) uuid is deprecated because it needed to be extended to 18 bytes.
+For more info, see this [MAVLink pull request](https://github.com/mavlink/mavlink/pull/786).
+Therefore, the discovery no longer contains the uuid.
+If the uid is required it can be found in the [Info plugin](../api_reference/classmavsdk_1_1_info.md#classmavsdk_1_1_info_1a812ed66265b7427bc781faec3f0fa89e).
```

Comment:
Is this link correct? ie have you updated the api ref. Couldn't see the UUID at the link

Reply:
It's supposed to go here: https://mavsdk.mavlink.io/develop/en/api_reference/classmavsdk_1_1_info.html#classmavsdk_1_1_info_1a812ed66265b7427bc781faec3f0fa89e

And then if you click on the actual struct https://mavsdk.mavlink.io/develop/en/api_reference/structmavsdk_1_1_info_1_1_identification.html there is a hardware_uid.

Reply:
Cool. Though it refers if you follow the links to uid2 of mavlink, and I have no idea what that is?